src/services/sync/strategies/consumerActions/UserAction.js

```diff
@@ -69,6 +70,18 @@ class UserAction extends BaseConsumerAction {
 		}
 	}
 
+	shouldSkipLdapSync(school) {
+		let skipLdapSync = false;
+		if (school.userLoginMigration) {
+			skipLdapSync =
+				school.userLoginMigration.closedAt ||
+				!school.features.includes(SCHOOL_FEATURES.ENABLE_LDAP_SYNC_DURING_MIGRATION);
+		} else {
+			skipLdapSync = !school.features.includes(SCHOOL_FEATURES.ENABLE_LDAP_SYNC_DURING_MIGRATION);
```

Comment:
The feature should not be evaluated if there is no user login migration.
The feature should only work together with the login migration.

Maybe we should rename the function to e.g. "skipLdapSyncWhileUserLoginMigration"